src/app/cli/src/explore/notebook_server_impl.rs

```diff
@@ -73,20 +73,20 @@ impl NotebookServerImpl {
             "Exposing Notebook server on a host network interface is not yet supported"
         );
 
-        let network_name = "kamu";
+        let network_name = get_random_name_with_prefix("kamu-");
 
         // Delete network if exists from previous run
         let _ = self
             .container_runtime
-            .remove_network_cmd(network_name)
+            .remove_network_cmd(&network_name)
             .stdout(std::process::Stdio::null())
             .stderr(std::process::Stdio::null())
             .output()
             .await;
 
         let network = self
             .container_runtime
-            .create_network(network_name)
+            .create_network(&network_name)
```

Comment:
Should we have `create_network_with_random_name()` method?

Reply:
We pass this in a few containers inside this method. So it probably should assigned once at the top and passed like that

Reply:
If the method returns `NetworkHandle` then we could extract the generated name from there. (Note `NetworkHandle` stores the name, but does not expose it currently, but I think it could).